### ISSUE TITLE
Feature: open raven in applet-section

### ISSUE THREAD

**Issue by arnegoeteyn** [budgie-desktop/budgie-desktop#762]
**Operating System**: Antergos
**Budgie version**: i.e. Budgie 10.2.9

Right now when I open raven with windows+n key shortcut it opens in the notification menu, even when there aren't any notifications. Could it be possible to make it open to the applet section when there aren't any notifications? Or a toggle to make it always open in the applet section?

**Comment by ikeydoherty** [comment]
Use Win+A - we already have this. :)

**Comment by arnegoeteyn** [comment]
Haha, my bad, thanks!

**Comment by ikeydoherty** [comment]
Not your fault - how was you to know? :D Discoverability currently sucks

**Comment by arnegoeteyn** [comment]
How I got to know about budgie? I saw some post on reddit about Solus and looked into that, I decided to try budgie.
I used to have a lot of problems with bluetooth on GNOME, my mouse would keep disconnecting, headset wouldn't pair in A2DP sync, etc.. I've only been using budgie for 2 days  but haven't encountered any of those problems, so that's enough to keep me from going back right now :p

Keep up the great work!
